website/docs/d/vra_region_enumeration_vsphere.html.markdown

```diff
@@ -0,0 +1,34 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration_vsphere"
+description: |-
+  Provides a data lookup for vra_region_enumeration_vsphere.
+---
+
+# Data Source: vra_region_enumeration_vsphere
+## Example Usages
+
+This is an example of how to read a region enumeration data source for vSphere cloud account.
+
+**Region enumeration vSphere data source by its id:**
+```hcl
+data "vra_region_enumeration_vsphere" "this" {
+  id = vra_region_enumeration_vsphere.this.id
+}
+```
+## Argument Reference
+
+* `id` - (Optional) The id of the region enumeration for vSphere account.
+
+* `dcId` - Identifier of a data collector vm deployed in the on premise infrastructure. Example: d5316b00-f3b8-4895-9e9a-c4b98649c2ca
+
+## Attribute Reference
+* `accept_self_signed_cert` - Accept self signed certificate when connecting to vSphere. Example: false
+
+* `hostname` - Host name for the cloud account endpoint. Example: dc1-lnd.mycompany.com
+
+* `password` - Password for the user used to authenticate with the cloud Account
+
+* `username` - Username to authenticate with the cloud account
+
```

Comment:
Same comment as above. Correct the example. Organize arguments and attributes correctly.